tests/integration_tests/functional/test_pause_resume.py

```diff
@@ -1,65 +1,112 @@
 
     # Resuming the microVM before being started is also not allowed.
-    response = test_microvm.vm.patch(state='Resumed')
-    assert test_microvm.api_session.is_status_bad_request(response.status_code)
+    response = microvm.vm.patch(state='Resumed')
+    assert microvm.api_session.is_status_bad_request(response.status_code)
 
-    # Start microVM.
-    test_microvm.start()
+    microvm.start()
 
-    ssh_connection = net_tools.SSHConnection(test_microvm.ssh_config)
+    ssh_connection = net_tools.SSHConnection(microvm.ssh_config)
 
     # Verify guest is active.
     exit_code, _, _ = ssh_connection.execute_command("ls")
     assert exit_code == 0
 
     # Pausing the microVM after it's been started is successful.
-    response = test_microvm.vm.patch(state='Paused')
-    assert test_microvm.api_session.is_status_no_content(response.status_code)
+    response = microvm.vm.patch(state='Paused')
+    assert microvm.api_session.is_status_no_content(response.status_code)
 
     # Verify guest is no longer active.
     exit_code, _, _ = ssh_connection.execute_command("ls")
     assert exit_code != 0
 
     # Pausing the microVM when it is already `Paused` is allowed
     # (microVM remains in `Paused` state).
-    response = test_microvm.vm.patch(state='Paused')
-    assert test_microvm.api_session.is_status_no_content(response.status_code)
+    response = microvm.vm.patch(state='Paused')
+    assert microvm.api_session.is_status_no_content(response.status_code)
 
     # Resuming the microVM is successful.
-    response = test_microvm.vm.patch(state='Resumed')
-    assert test_microvm.api_session.is_status_no_content(response.status_code)
+    response = microvm.vm.patch(state='Resumed')
+    assert microvm.api_session.is_status_no_content(response.status_code)
 
     # Verify guest is active again.
     exit_code, _, _ = ssh_connection.execute_command("ls")
     assert exit_code == 0
 
     # Resuming the microVM when it is already `Resumed` is allowed
     # (microVM remains in the running state).
-    response = test_microvm.vm.patch(state='Resumed')
-    assert test_microvm.api_session.is_status_no_content(response.status_code)
+    response = microvm.vm.patch(state='Resumed')
+    assert microvm.api_session.is_status_no_content(response.status_code)
 
     # Verify guest is still active.
     exit_code, _, _ = ssh_connection.execute_command("ls")
     assert exit_code == 0
+
+    microvm.kill()
+    del tap
+
+
+@pytest.mark.skipif(
+    platform.machine() != "x86_64",
+    reason="Not supported yet."
+)
+def test_pause_resume(test_session_root_path,
+                      network_config,
+                      bin_cloner_path):
+    """Test scenario: boot/pause/resume for all available configurations."""
+    logger = logging.getLogger("pause_resume")
+    # Currently, artifacts share the bucket with all other resources.
+    artifact_collection = ArtifactCollection(test_images_s3_bucket())
```

Comment:
Would be nice to add a comment here why we have that `integration_tests/functional/test_pause_resume.py::test_images_s3_bucket` in the `tools/devtool test` output (in case you didn't fix that meantime).

Reply:
I've reverted back to `from conftest import init_microvm`. Seems to work now, maybe there was something wrong locally.

The PR does not fail on unused checks, neither is my IDE. It is used as it is reference by the delete.